configure.ac

```diff
@@ -141,12 +141,19 @@ AC_DEFINE_UNQUOTED([VA_DRIVER_INIT_FUNC], [$VA_DRIVER_INIT_FUNC],
     [Define driver entry-point])
 
 dnl Check for VA/DRM API
-USE_X11="$enable_x11"
-if test "$USE_X11" = "yes"; then
-    PKG_CHECK_MODULES(LIBVA_X11_DEPS, [libva-x11],
-      [AC_DEFINE([HAVE_VA_X11], [1], [Defined to 1 if VA/X11 API is enabled])],
-      [USE_X11="no"])
+USE_X11="no"
+if test "$enable_x11" = "yes"; then
```

Comment:
@seanvk, a user should install the develop package for X11, otherwise it will break the compilation although user installs the develop packages for libva.